services/ui-auth/serverless.yml

```diff
@@ -28,6 +28,8 @@ custom:
   sesSourceEmailAddress: ${ssm:/configuration/${self:custom.stage}/sesSourceEmailAddress~true, ssm:/configuration/default/sesSourceEmailAddress~true, ""}
   attachments_bucket_arn: ${cf:uploads-${self:custom.stage}.AttachmentsBucketArn}
   api_gateway_rest_api_name: ${cf:app-api-${self:custom.stage}.ApiGatewayRestApiName}
+  okta_metadata_url: ${ssm:/configuration/${self:custom.stage}/okta_metadata_url~true, ""}
```

Comment:
This is the variable that indicates whether or not Okta support should be built.  If a branch specific SSM parameter exists, Okta support is built